### PR TITLE
support "psr/http-message" v2 version

### PR DESCRIPTION
**Pull request by gaalferov**
## Motivation
Support for the new version of the PSR [PHP Standards Recommendations] (no breaking changes)

## Changes

* Add support of psr/http-message v2 
* Run tests with Symfony framework (GitHub actions) 

## How to test
```
composer test
```

### CODE REVIEW

File: .github/workflows/ci-phpunit.yml [hunk 1]
```diff
@@ -11,11 +11,12 @@ on:
 
 jobs:
   phpunit-tests:
+    name: PHP ${{ matrix.php }} Latest
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
         os: [ ubuntu-latest ]
-        php-versions: [ '7.4','8.0','8.1','8.2' ]
+        php: [ '7.4','8.0','8.1','8.2' ]
```
Comment (IgorDobryn) [inline .github/workflows/ci-phpunit.yml:19]: Slight preference for consistency:
```suggestion
        php-version: [ '7.4','8.0','8.1','8.2' ]
```

File: .github/workflows/ci-phpunit.yml [hunk 2]
```diff
@@ -50,3 +51,35 @@ jobs:
...
       - name: Run phpunit tests
         run: composer test
+
+  symfony:
+    name: Symfony ${{ matrix.symfony }} LTS
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - symfony: '5'
```
Comment (IgorDobryn) [inline .github/workflows/ci-phpunit.yml:61]: For consistency:
```suggestion
          - symfony-version: '5'
```

**Review by IgorDobryn — APPROVED** [review]

File: .github/workflows/ci-phpunit.yml (hunk 2 above)
Comment (gaalferov) [reply .github/workflows/ci-phpunit.yml:61]: This is the strict variable name for the "matrix" action

File: .github/workflows/ci-phpunit.yml (hunk 1 above)
Comment (gaalferov) [reply .github/workflows/ci-phpunit.yml:19]: Done

**Review by VladimirTaytor — APPROVED** [review]

**Pull request merged**